### PR TITLE
ADDED DARK MODE functionality

### PR DESCRIPTION
**Pull request by abhinav2712**
### **Description**
This pull request adds dark mode functionality to the web page. When the user clicks on the dark mode button, the page background color and text color will be changed to make the page easier to view in low-light conditions. The user's choice of light or dark mode is stored in the browser's local storage, so the page will remember the user's preference across sessions.

### **Changes Made**

- Added a "Dark Mode" button to the page header.
- Implemented a toggle function using JavaScript to change the background color, text color, and button icon when the user switches between light and dark mode.
- Added CSS styles for the dark mode theme.

### **Testing**
I have tested the dark mode feature on various browsers and devices to ensure compatibility and functionality.

### **Checklist**
 

- [x] Tested the dark mode feature on various browsers and devices
- [x]  Reviewed the code for readability and maintainability
- [x]  Ensured the feature meets the requirements outlined in the project brief
- [ ]  Added appropriate comments to the code
- [ ]  Made the button more with the theme design

Thank you for taking your time to go through my PR, I hope to get relevant comments and improvements
Happy Hacking!

### CODE REVIEW

**Comment by Snailedlt** [comment]
Could you please rebase on develop so that we don't get those extra merge commits? :)

**Comment by abhinav2712** [comment]
@Snailedlt I need some assistance in this, Like to rebase I have to do ``` git checkout develop ``` in local , but after forking, i dont have any branches of the original repo hence once I checkout , it shows develop doesnt exist in any files. 

Or do i have to clone the original repo then do checkout instead of forking and cloning? 
Also if I go with second approach, shouldnt that be not allowed since i cant push / PR into orginal repo right?

**Comment by Snailedlt** [comment]
@abhinav2712 ahh yes! I keep forgetting that GitHub [changed forking](https://github.blog/changelog/2022-07-27-you-can-now-fork-a-repo-and-copy-only-the-default-branch/) some time ago. Unless you uncheck the `only for main branch` checkbox it will only fork that branch, instead of all branches. It should be possible to go to branches in the original repo and fork from there I think.

After you have develop in your GitHub fork, you can do the following to rebase on develop:
```sh
git checkout develop
git pull
git checkout your_branchname
git rebase develop
```

File: docs/assets/css/style.css [hunk 1]
```diff
@@ -464,3 +464,67 @@ input[type='color']:hover {
...
+      color:black;
+      display: inline;  
+  }
+
+  .darkbutton .image{
+     width:30px;
+     height:30px;    
+     display: inline;  
+    
+}
```
Comment (Panquesito7) [inline docs/assets/css/style.css:530]:
```suggestion
}

```

**Comment by Panquesito7** [comment]
Could you please show us a screenshot of how it looks? Thanks. 🙂

**Comment by abhinav2712** [comment]
@Panquesito7 Just made some changes and added a new PR, check it, that has the screenshot and let me know if its acceptable

**Comment by Snailedlt** [comment]
@abhinav2712 does that mean we can close this PR?

**Pull request closed without merging**